packages/app-builder-lib/templates/nsis/include/installUtil.nsh

```diff
@@ -126,7 +126,10 @@ Function handleUninstallResult
   Return
 
   ${if} $R0 != 0
+    MessageBox MB_OK|MB_ICONEXCLAMATION "$(uninstallFailed): $R0"
     DetailPrint `Uninstall was not successful. Uninstaller error code: $R0.`
+    SetErrorLevel 2
+    Quit
```

Comment:
I think ideally we should retry running uninstaller (with user's confirmation) in such case instead of failing whole installation immediately. Let me know if you want me to add this to the current patch.

Reply:
Oh, and btw. `Abort` doesn't quite work here for some reason (the installer just hangs), so I had to resort to `Quit`.

Reply:
@indutny-signal This sounds like a great idea. Is it complex to add?
>we should retry running uninstaller (with user's confirmation) in such case instead of failing whole installation immediately

Reply:
I actually already pushed it :wink: